### PR TITLE
[16.0] [MIG] account_payment_partner

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
https://github.com/OCA/bank-payment/issues/964

Needs https://github.com/OCA/bank-payment/pull/968

### CODE REVIEW

**Comment by marylla** [comment]
> Needs #968

@ThomasBinsfeld PR #968 is merged now. Could you please rebase this PR? I'd like to test it in runboat.

File: account_payment_partner/views/account_move_line.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
         <field name="inherit_id" ref="account.view_move_line_form" />
         <field name="arch" type="xml">
             <xpath
-                expr="//group[@groups='analytic.group_analytic_accounting,analytic.group_analytic_tags']"
+                expr="//field[@name='analytic_distribution_stored_char']/.."
```
Comment (atchuthan) [inline account_payment_partner/views/account_move_line.xml]: Getting error with this Xpath
Change it to 
```
            <xpath
                expr="//group[@groups='analytic.group_analytic_accounting']"
                position="after"
            >
```

Error detail
```
Element '<xpath expr="//field[@name=&#39;analytic_distribution_stored_char&#39;]/..">' cannot be located in parent view

View error context:
{'file': '/opt/odoo/appstest16-odoo/src/OCA/bank-payment/account_payment_partner/views/account_move_line.xml',
 'line': 2,
 'name': 'account_payment_partner.move_line_form',
 'view': ir.ui.view(2118,),
 'view.model': 'account.move.line',
 'view.parent': ir.ui.view(568,),
 'xmlid': 'view_move_line_form'}
```

**Review by atchuthan — CHANGES_REQUESTED** [review]
Except the view error, other functionalities tested :+1:

**Comment by etobella** [comment]
@ThomasBinsfeld can you rebase?

**Comment by ThomasBinsfeld** [comment]
> @ThomasBinsfeld can you rebase?

Done

**Comment by etobella** [comment]
Tests are failing due to some problems with a view. Can you check it?

**Comment by etobella** [comment]
This should fix pre-commit #989

**Comment by florian-dacosta** [comment]
Hello @ThomasBinsfeld could you fix the view issue and make a new rebase ?
If needed, I can superseed this PR to make the changes, you let me know

**Comment by ThomasBinsfeld** [comment]
> Hello @ThomasBinsfeld could you fix the view issue and make a new rebase ? If needed, I can superseed this PR to make the changes, you let me know

Fixed

**Review by pere-aquarian — APPROVED** [review]

**Review by atchuthan — APPROVED** [review]

**Pull request merged**